### PR TITLE
address storage feedback

### PR DESCRIPTION
**Pull request by hdworld11**
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] Swift
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx [hunk 1]
```diff
@@ -68,10 +66,6 @@ Learn more about how you can further customize the UI component by referring to
 
 ## API Usage
```
Comment (aspittel) [inline src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx:67]: I would still retitle this header - Implement Uploads for ex.

**Review by aspittel — APPROVED** [review]

File: src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx [hunk 2]
```diff
@@ -112,9 +109,38 @@ function App() {
     );
 }
 ```
+</InlineFilter>
+
+<InlineFilter filters={["javascript"]}>
```
[the remark this answers is not in the document]
Comment (hdworld11) [reply src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx:114]: Yes, updated

File: src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx (hunk 1 above)
Comment (hdworld11) [reply src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx:67]: Changed to `Implement upload functionality`

File: src/pages/[platform]/build-a-backend/storage/set-up-storage/index.mdx [hunk 3]
```diff
@@ -32,21 +32,17 @@ In this guide, you will learn how to set up storage in your Amplify app. You wil
 
 If you have not yet created an Amplify app, visit the [quickstart guide](/[platform]/start/quickstart/).
 
-{/* Let's add a description of what Storage is and why you would want to use it in your app. Not sure if this is how we want to message it  */}
-
-With Amplify Storage, you can enable file storage and management for user's content in your app. Paths within your storage instance can have different access rules. Amplify Storage is powered by Amazon Simple Storage Service (Amazon S3) buckets.
+Amplify Storage seamlessly integrates file storage and management capabilities into frontend web and mobile apps, built on top of Amazon S3. It provides intuitive APIs and UI components for core file operations, enabling developers to build scalable and secure file storage solutions without dealing with cloud service complexities.
```
[the remark this answers is not in the document]
Comment (hdworld11) [reply src/pages/[platform]/build-a-backend/storage/set-up-storage/index.mdx:35]: Updated

**Review by aspittel — APPROVED** [review]

**Pull request merged**